proguard-android-optimize.txt

```diff
@@ -20,19 +20,3 @@
 -keepclassmembers class * implements android.os.Parcelable {
   public static final ** CREATOR;
 }
-
--keep class androidx.annotation.Keep
-
--keep @androidx.annotation.Keep class * {*;}
-
--keepclasseswithmembers class * {
-    @androidx.annotation.Keep <methods>;
-}
-
--keepclasseswithmembers class * {
-    @androidx.annotation.Keep <fields>;
-}
-
--keepclasseswithmembers class * {
-    @androidx.annotation.Keep <init>(...);
-}
```

Comment:
This should not be removed, it's required to preserve reflectively accessed fields and classes.

Reply:
But the `annotaion` lib should already contains this

```
-keep,allowobfuscation @interface androidx.annotation.Keep
-keep @androidx.annotation.Keep class * {*;}

-keepclasseswithmembers class * {
    @androidx.annotation.Keep <methods>;
}

-keepclasseswithmembers class * {
    @androidx.annotation.Keep <fields>;
}

-keepclasseswithmembers class * {
    @androidx.annotation.Keep <init>(...);
}

-keepclassmembers,allowobfuscation class * {
  @androidx.annotation.DoNotInline <methods>;
}

```

Reply:
https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-main/annotation/annotation/src/main/resources/META-INF/proguard/androidx-annotations.pro?source=post_page---------------------------%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F&autodive=0

Reply:
That's fine then. I recommend sharing your reasoning for non-obvious changes within the commit message itself than requiring a reviewer to ask a question.

Reply:
Will do.